src/standalones/clearOrphanedRoles.ts

```diff
@@ -0,0 +1,15 @@
+import { Guild } from 'discord.js'
+
+export async function clearOrphanedRoles(guild: Guild) {
+  const hexExp = /^#?[0-9A-F]{6}$/i
+
+  let rolesDeleted = 0
+  guild.roles.cache
+    .filter((role) => hexExp.test(role.name))
```

Comment:
Wouldn't it be better to check two conditions with one iteration of filter?

Reply:
Good call, fixed